__tests__/app.test.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const request = require('supertest')
+const app = require('../app')
+
+beforeAll(() => seed(data));
+afterAll(() => db.end)
+
```

Comment:
we can also have a test for when the user tries to access an non-existent endpoint